blog/2021-11-18-pants-vs-bazel/index.mdx

```diff
@@ -54,6 +54,7 @@ We have a lot of respect for Bazel. Pants and Bazel share common lineage and ins
 Here are some reasons why Pants may be a better choice than Bazel for your team:
 
 - **Pants is strongly focused on first-class support for the real-world use cases of a variety of software organizations.** Bazel, in contrast, was designed primarily for internal C++ usage at Google. For example:
+
```

Comment:
Drive-by lint warning that got fixed while formatting the other file.

Reply:
The hell? This causes the check to fail in CI but is required for the check to pass locally.